pkg/plugins/noderesourcetopology/binder.go

```diff
@@ -0,0 +1,65 @@
+package noderesourcetopology
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	jsonpatch "github.com/evanphx/json-patch"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/klog/v2"
+	"k8s.io/kubernetes/pkg/scheduler/framework"
+
+	topologyv1alpha1 "github.com/gocrane/api/topology/v1alpha1"
+)
+
+// PreBind writes pod topology result annotations using the k8s client.
+func (tm *TopologyMatch) PreBind(
+	ctx context.Context,
+	state *framework.CycleState,
+	pod *corev1.Pod,
+	nodeName string,
+) *framework.Status {
+	klog.V(3).InfoS("Attempting to prebind pod to node", "pod", klog.KObj(pod), "node", nodeName)
```

Comment:
dynamic plugin is using 2 and 4 as log level, should nrt plugin match the same level?

Reply:
Sure.